src/components/svg-canvas-graph.tsx

```diff
@@ -278,6 +365,18 @@ const SvgCanvas = () => {
                     handlePointerDown={() => {}} // no use
                 />
             )}
+            {activePolylines.length !== 0 &&
+                activePolylines.map((p, i) => (
+                    <path
+                        key={`line_polyline_${i}`}
```

Comment:
We should use the a,b,c,x,y in p instead of i

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/